.travis.yml

```diff
@@ -1,6 +1,13 @@
 language: node_js
 before_install:
   - npm install -g npm@latest
+install:
+  - npm install --global electron
```

Comment:
As far as I know Electron is tracking latest stable node (currently 7.x):
```
> process.versions.node
< "7.4.0"
```
The different version thing is the v8 ABI, which usually is only of concern for native bindings.